build.sbt

```diff
@@ -61,10 +61,11 @@ lazy val loader = project
       Dependencies.directRunner,
       Dependencies.metrics,
       Dependencies.scioCore,
-      Dependencies.scioBigQuery,
+      Dependencies.scioGoogle,
       Dependencies.scioTest,
       Dependencies.fastjson
-    )
+    ),
+    resolvers += "Confluent Repository".at("https://packages.confluent.io/maven/")
```

Comment:
I would add it here: https://github.com/snowplow-incubator/snowplow-bigquery-loader/blob/master/project/BuildSettings.scala#L115